provider/openstack/networking.go

```diff
@@ -147,6 +147,14 @@ type NeutronNetworking struct {
 	networkingBase
 }
 
+// networkFilter returns a neutron.Filter to match Neutron Networks with
+// the exact given name AND router:external boolean result.
+func projectIdFilter(projectId string) *neutron.Filter {
+	filter := neutron.NewFilter()
+	filter.Set(neutron.FilterProjectId, fmt.Sprintf("%s", projectId))
```

Comment:
projectId is already a string, no need for fmt.Sprintf

Reply:
updated.